packages/schema-dot-org-types/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "schema-dot-org-types",
+  "version": "0.1.0",
+  "description": "Classes with flow types to encapsulate schema.org markup attributes.",
+  "main": "index.js",
+  "scripts": {
+    "build": "npm run test && flow && rm -rf ./lib && babel src -d lib && babel tools -d bin && rollup -c -i src/index.js && flow-copy-source src lib",
```

Comment:
This chain is way, wayyyyy too long. `npm test` should be called from the root, there's no need for it in this script. `prebuild` and `postbuild` are available, too, when `npm run build` is called.